pkgs/tools/audio/beets/common.nix

```diff
@@ -25,6 +25,7 @@
 , version
 , extraPatches ? [ ]
 , pluginOverrides ? { }
+, deprecatedPlugins ? [ "absubmit" ]
```

Comment:
Is there a reason why anyone would want to override this? If no, I'd push that one down inside the `let`.

Reply:
> Is there a reason why anyone would want to override this? If no, I'd push that one down inside the `let`.

Maybe that's too much, but what I was thinking about was that someone may wish to launch a absubmit server locally and point the plugin to their local instance, and then they would want to use this plugin, and override this list.

If you think I went to far I'll remove it.

Reply:
> If you think I went to far I'll remove it.

Irrelevant dilemma now